packages/dom-helper/lib/main.js

```diff
@@ -242,6 +242,14 @@ if (canRemoveSvgViewBoxAttribute){
 }
 
 prototype.setPropertyStrict = function(element, name, value) {
+  if (value === undefined) {
+    value = null;
+  }
+
+  if (value === null && (name === 'value' || name === 'type' || name === 'src')) {
```

Comment:
Isn't this dependent on element type?  Should we check and cache to make sure we do the right thing here?